### ISSUE TITLE
Update PRoot section in README.md file

### ISSUE THREAD

**Issue by oxr463** [fsquillace/junest#278]
>  One of the major drawbacks is the fact that Proot is not officially supported anymore, therefore, Proot bugs may no longer be fixed.

Source: https://github.com/fsquillace/junest#proot-based

This isn't quite accurate anymore.

It would be great to see how JuNest runs with the latest version of PRoot:

- https://github.com/proot-me/proot/releases/tag/v5.2.0

We even have a rust port from GSoC 2021:

- https://github.com/proot-me/proot-rs/releases/tag/v0.1.0

**Comment by fsquillace** [comment]
Thanks @oxr463 !

I updated the JuNest AUR package repository here: https://github.com/fsquillace/junest-aur-repo/blob/master/pkgs/proot-static/PKGBUILD

which, in turn, produces the compiled packages here: https://github.com/fsquillace/junest-repo/tree/main/any


You should be able to update the repo of an existing JuNest environment with:

```
sudo pacman -Sy proot
```

I will update both the doc and the JuNest image later on.

**Comment by fsquillace** [comment]
https://github.com/fsquillace/junest/pull/280

**Comment by fsquillace** [comment]
Very interesting there is a rust port for proot! This may bring proot to a even brighter future. I created this issue to explore usage of it under JuNest: https://github.com/fsquillace/junest/issues/281

It is possible to use any custom proot backend with the `--backend-command` option. @oxr463 in case you are interested to collaborate to this project, you are more than welcome :)

Thanks again!
Closing this issue.
